### DISCUSSION TITLE
meshcentral from intel kind of abandoned

### DISCUSSION THREAD

**Discussion by lazee486** [amidaware/tacticalrmm#1658] [General]
**Is your feature request related to a problem? Please describe.**
default install is with a tool that may or may not continue...

**Describe the solution you'd like**
a choice when installing for another remote control backend, and/or a migration method.

**Describe alternatives you've considered**
I see it does already support some others like rustdesk....who havent decided what is commercial what isnt...Id like to go with Tacticals preffered solution.

**Additional context**
https://meshcentral2.blogspot.com/2023/02/starting-work-at-microsoft.html

We are trying to leave teamviewer and this was looking SO good :) work was even fine with paying for macos signing...

just want clarity because Ive been pushing for this....

**Comment by dinger1986** [comment]
As you said there’s loads of options.

Meshcentral released a new version last week or the week before, it is in no way abandoned. That statement says it’s no longer a priority but that’s mainly because the main developers aren’t being paid to develop it anymore so it’s the same as a lot or opensource projects. The community is active and there is plenty of people who would take over if it was abandoned.

We have already added to our docs and are committed to ensuring mesh is secure and stable https://docs.tacticalrmm.com/mesh_integration/#omg-meshcentral-isnt-maintained-anymore
